macosx/douninstall.sh

```diff
@@ -6,7 +6,10 @@ PREFIX="/usr/local"
 BINDIR="$PREFIX/bin"
 PKGCONFIGDIR="$PREFIX/lib/pkgconfig"
 GREINCLUDEDIR="$PREFIX/include/gregorio"
-TEXMFLOCAL=`/usr/texbin/kpsewhich -var-value TEXMFLOCAL`
+TEXMFLOCAL=`kpsewhich -var-value TEXMFLOCAL`
+if [ -z "$TEXMFLOCAL" ]; then
+    TEXMFLOCAL=`/usr/texbin/kpsewhich -var-value TEXMFLOCAL`
+fi
```

Comment:
Is it possible TEXMFLOCAL will have a path in it (a la #235)? If so, it might be better to use `kpsewhich gregoriotex.tex` to find the location of the things to delete.